pkg/envoy/cds/cluster.go

```diff
@@ -75,16 +75,6 @@ func getOutboundPassthroughCluster() *xds_cluster.Cluster {
 }
 
 // getSyntheticCluster returns a static cluster with no endpoints
```

Comment:
@adithyaakrishna it looks like we can remove this comment as well!
```suggestion
```

Reply:
My bad, I missed this. Will fix it ASAP